decred/tests/unit/dcr/test_vsp_unit.py

```diff
@@ -34,7 +34,9 @@ def __init__(
 purchaseInfo = {
     "PoolAddress": "TsbyH2p611jSWnvUAq3erSsRYnCxBg3nT2S",
     "PoolFees": 0.5,
-    "Script": "512103af3c24d005ca8b755e7167617f3a5b4c60a65f8318a7fcd1b0cacb1abd2a97fc21027b81bc16954e28adb832248140eb58bedb6078ae5f4dabf21fde5a8ab7135cb652ae",
+    "Script": "512103af3c24d005ca8b755e7167617f3a5b4c60a65f8318a7fcd1b0cacb1ab"
+    + "d2a97fc21027b81bc16954e28adb832248140eb58bedb6078ae5f4dabf21fde5a8ab713"
```

Comment:
You don't need the addition here, that would be done at runtime. Instead you just put the strings one after the other, and the interpreter will collapse them into one at bytecode-generation time. That's implicit string concatenation: implicit because there's no operation indicated. From [the docs](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str):

"String literals that are part of a single expression and have only whitespace between them will be implicitly converted to a single string literal. That is, ("spam " "eggs") == "spam eggs".

Reply:
wow.